controller/DeliveryMgmt.php

```diff
@@ -115,7 +117,7 @@ public function editDelivery()
         }
         
         // define the groups related to the current delivery
-        $property = new core_kernel_classes_Property(PROPERTY_GROUP_DELVIERY);
+        $property = new core_kernel_classes_Property(GroupAssignment::GROUP_DELIVERY);
```

Comment:
as it is a property you should keep consistency with other constant name (always put PROP or PROPERTY in the name)

Reply:
Ok, good to know, I will change this also in other PR's regarding the same topic